src/ui.form-input-search/config.ts

```diff
@@ -2,7 +2,7 @@ export default /*tw*/ {
   searchInput: "{UInput}",
   clearIcon: "{UIcon}",
   searchIcon: "{UIcon}",
-  searchButton: "{UButton} rounded-l-none ml-1 outline outline-1 outline-gray-900",
+  searchButton: "{UButton} -mr-3 rounded-l-none ml-1 outline outline-1 outline-gray-900",
```

Comment:
no, you need to add mutatedProp and cva variant

Reply:
Cva variant for what?

Reply:
`-mr-3` is bad solution. You need to remove right slot padding, if search button label is not empty